public/admin/config.yml

```diff
@@ -668,5 +668,10 @@ collections:
         widget: "string"
         required: false
         hint: "External link to the event page"
+      - name: "luma-calendar-link"
+        label: "Luma Calendar URL"
+        widget: "string"
+        required: false
+        hint: "Luma Calendar URL for the event. Include relevant tags when appropriate, e.g., https://lu.ma/filecoin?tag=fil%20bangkok"
```

Comment:
Just curious - why adding the tag? We want to have the title of the event as a tag? Do they edit the url manually?